README.md

```diff
@@ -7,10 +7,11 @@ iocage is a jail/container manager amalgamating some of the best features and
 technologies the FreeBSD operating system has to offer. It is geared for ease
  of use with a simple and easy to understand command syntax.
 
-iocage is in the FreeBSD ports tree as sysutils/iocage.
-To install using binary packages, simply run: `pkg install iocage`
+iocage is in the FreeBSD ports tree as sysutils/py-iocage.
+To install using binary packages, simply run: `sysutils/py-iocage`
```

Comment:
Yeah all python ports do that so we can have Python 2 and 3 ports coexist. It's a fair trade off. I just committed and gave you credit in title. Thanks again!